presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java

```diff
@@ -1363,6 +1363,20 @@ public static long widthBucket(@SqlType(StandardTypes.DOUBLE) double operand, @S
         return lower;
     }
 
+    @Description("The bucket number of a value given an array of bins")
```

Comment:
Could you make it a more precise description to incorporate bin names?

Reply:
Fixed.